Tests/SessionTests.swift

```diff
@@ -815,17 +815,20 @@ final class SessionTestCase: BaseTestCase {
 
     // MARK: Tests - Request Retrier
 
-    func testThatSessionCallsRequestRetrierWhenRequestEncountersError() {
+    func testThatSessionCallsRequestRetrierWhenRequestInitiallyEncountersAdaptError() {
```

Comment:
No tests were changed, I just moved some around so all of the retrier tests could be together so they can be disabled.